app/javascript/angular/dashboard/widget-wrapper.js

```diff
@@ -68,11 +87,11 @@ ManageIQ.angular.app.component('widgetWrapper', {
         </div>
         <widget-error ng-if="vm.error === true"></widget-error>
         <widget-spinner ng-if="!vm.widgetModel && vm.widgetBlank == 'false' && !vm.error"></widget-spinner>
-        <div ng-if="vm.widgetBlank === 'true' || vm.widgetModel"
+        <div ng-if="vm.widgetModel.blank === 'true' || vm.widgetModel"
```

Comment:
this no longer makes sense

the or part is always true if widgetModel.blank wasn't an error

Reply:
fixed